iothub/device/tests/IotHubDeviceClientTests.cs

```diff
@@ -11,17 +11,21 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Azure.Devices.Client.Test
 {
     [TestClass]
     [TestCategory("Unit")]
     public sealed class IotHubDeviceClientTests : IDisposable
     {
-        private const string FakeConnectionString = "HostName=acme.azure-devices.net;SharedAccessKeyName=AllAccessKey;DeviceId=fake;SharedAccessKey=dGVzdFN0cmluZzE=";
-        private const string FakeHostName = "acme.azure-devices.net";
+        static readonly string fakeHostName = "acme.azure-devices.net";
+        static readonly string fakeDeviceId = "fake";
+        static readonly string fakeSharedAccessKey = "dGVzdFN0cmluZzE=";
+        static readonly string fakeSharedAccessKeyName = "AllAccessKey";
+        static readonly string fakeConnectionString = $"HostName={fakeHostName};SharedAccessKeyName={fakeSharedAccessKeyName};DeviceId={fakeDeviceId};SharedAccessKey={fakeSharedAccessKey}";
```

Comment:
Always specify visibility.

```suggestion
        private static readonly string fakeHostName = "acme.azure-devices.net";
        private static readonly string fakeDeviceId = "fake";
        private static readonly string fakeSharedAccessKey = "dGVzdFN0cmluZzE=";
        private static readonly string fakeSharedAccessKeyName = "AllAccessKey";
        private static readonly string fakeConnectionString = $"HostName={fakeHostName};SharedAccessKeyName={fakeSharedAccessKeyName};DeviceId={fakeDeviceId};SharedAccessKey={fakeSharedAccessKey}";
```